sources/packages/backend/libs/sims-db/src/entities/application-restriction-bypass.model.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from "typeorm";
+import { ColumnNames, TableNames } from "../constant";
+import { RecordDataModel } from "./record.model";
+import {
+  User,
+  Note,
+  StudentRestriction,
+  RestrictionBypassBehaviors,
+  Application,
+} from "@sims/sims-db";
+
+/**
+ * Restrictions bypass that allow awards to be disbursed ignoring
+ * restrictions at the student application level.
+ */
+@Entity({ name: TableNames.ApplicationRestrictionBypasses })
+export class ApplicationRestrictionBypass extends RecordDataModel {
+  /**
+   * Auto-generated sequential primary key column.
+   */
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  /**
+   * Reference to the student application that will have the bypass applied.
+   */
+  @ManyToOne(() => Application, {
+    eager: false,
+    cascade: ["update"],
+  })
+  @JoinColumn({
+    name: "application_id",
+    referencedColumnName: ColumnNames.ID,
+  })
+  application: Application;
+
+  /**
+   * Active student restriction to be bypassed.
+   */
+  @ManyToOne(() => StudentRestriction, {
+    eager: false,
+    cascade: ["update"],
```

Comment:
https://github.com/bcgov/SIMS/pull/3671/files#r1742823044

Reply:
Not right now, it can be defined as false.

Reply:
Set to`false`.